proc/proc_windows.go

```diff
@@ -62,6 +59,32 @@ func Launch(cmd []string) (*Process, error) {
 		defer syscall.CloseHandle(syscall.Handle(fd[i]))
 	}
 
+	argv0, err := syscall.UTF16PtrFromString(argv0Go)
+	if err != nil {
+		return nil, err
+	}
+
+	// create suitable command line for CreateProcess
+	// see https://github.com/golang/go/blob/master/src/syscall/exec_windows.go#L326
+	// adapted from standard library makeCmdLine
+	// see https://github.com/golang/go/blob/master/src/syscall/exec_windows.go#L86
+	var cmdLineGo string
+	if len(cmd) > 1 {
```

Comment:
Sorry I didn't catch this in earlier reviews, but this branch seems like an error. If we enter the branch, we call `syscall.EscapeArg` on every element in the `cmd` slice, including index `0`. However if we don't enter that branch, index `0` never gets escaped.

Unless I'm mistaken it seems like we would want to escape everything the `cmd` slice regardless of the number of elements, or at least if `len(cmd) > 0`.